source/agora/test/Base.d

```diff
@@ -540,6 +540,9 @@ public class TestAPIManager
         heights for only these nodes. Note that the clock time is adjusted
         for all nodes (this is what most tests expect).
 
+        Other overloads allow passing a `BlockHeader` of an enrolled height to
+        get enrollments and a height for checking pre-images are revealed.
```

Comment:
This seems to me like an abuse of overloading. The other overloads do much more (checking pre-images) than this. Perhaps the other overloads should just be the default ? Although the client doesn't care much about pre-images.

Reply:
I'm thinking about what the best document for that is. I'll add right description.